lib/elixir_analyzer/exercise_test/common_checks/exemplar_comparison.ex

```diff
@@ -7,10 +7,10 @@ defmodule ElixirAnalyzer.ExerciseTest.CommonChecks.ExemplarComparison do
   alias ElixirAnalyzer.Constants
   alias ElixirAnalyzer.Comment
 
-  @spec run(Macro.t(), atom, Macro.t()) :: [{:pass | :fail | :skip, %Comment{}}]
+  @spec run(Macro.t(), atom, Macro.t()) :: [{:pass | :fail | :skip, Comment.t()}]
 
   def run(code_ast, :concept, exemplar_ast) do
-    if Macro.to_string(code_ast) == Macro.to_string(exemplar_ast) do
```

Comment:
Did using `Macro.to_string` cause issues?

Reply:
Yes, the exemplar comparison for captains-log and boutique-inventory started failing because of differences in formatting.